### DISCUSSION TITLE
SQLSTATE[HY000]: General error: 8 attempt to write a readonly database ; .htaccess

### DISCUSSION THREAD

**Discussion by byChUcK** [UnamSanctam/UnamWebPanel#34] [General]
After correcting an error in the creation of the Logs directory, I have this db error return message, the database is with privileges 777, I'm hosting on an apache server https://prnt.sc/24l60vg .

The .htaccess file is not blocking access to my db.

Please what should I do to fix the errors?

Congratulations and thanks for sharing your projects and knowledge =D

**Comment by UnamSanctam** [comment]
Make sure that the folder that the database is in has write privileges as well.

**Reply by byChUcK** [reply]
The db file is in the root folder of www/html, html folder with privilege 775, it keeps giving the same error! https://prnt.sc/24na31q

**Reply by UnamSanctam** [reply]
And the ownership is correct and everything?

**Reply by byChUcK** [reply]
I think so, could you add me on discord or telegram or pidgin? to send you help me directly accessing vps thank you!

tele: by_chuckpriv8
discord: by_ChUcK#3913
pidgin: chuckroot@jabster.pl

**Reply by byChUcK** [reply]
Hello bro, I managed to solve the db problem, I'm just not able to make the db inaccessible even by putting the .htaccess I can download the db externally via the link, do you know how to proceed?
![Screenshot_3](https://user-images.githubusercontent.com/96556095/147526837-3f5464f8-1da2-4ebe-b490-6dac4e45a6f4.jpg)
![Screenshot_4](https://user-images.githubusercontent.com/96556095/147526839-36822808-ac06-4a63-8b17-6be44279c3b1.jpg)

**Reply by UnamSanctam** [reply]
Ah, sorry for not responding earlier, I missed the notification. But try editing `/etc/apache2/apache2.conf` and make sure that the line `AccessFileName .htaccess` is uncommented (no # in front of it).

**Reply by byChUcK** [reply]
Thanks for answering!

The comment in the apache2.conf file is already without the "#" in the .htaccess line, what content do you put inside the .htaccess file?
![Screenshot_5](https://user-images.githubusercontent.com/96556095/147588876-ab5ed297-2806-4a40-8ad3-5406ac01552e.jpg)

**Reply by UnamSanctam** [reply]
It contains:
```
<Files "unamwebpanel.db">  
  Require all denied
</Files>
```

**Reply by byChUcK** [reply]
Would UnamSanctam your tools have private version? if yes I'm interested in buying, if I don't have some suggestions that can be inserted for an even better experience, congratulations for the tools =D

**Reply by byChUcK** [reply]
> Would UnamSanctam your tools have private version? if yes I'm interested in buying, if I don't have some suggestions that can be inserted for an even better experience, congratulations for the tools =D

hello

**Reply by UnamSanctam** [reply]
Hey, no I don't have any private or paid versions, I keep everything public and open source.
